tests/jsTests/getHelpNowSpec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Suite : Tests for show get help action section
+ * Description : This is a test suite for checking if the get-help-action section (call/sms) is visible
+ */
+describe("Tests for show get help action section", function() {
+	beforeEach(function() {
+		jasmine.clock().install();
+		setFixtures('<div id="get-help-section"></div> \
+				<div id="get-help-action-section"></div>');
+		spyOn($.fn, "fadeOut").and.callFake(function() {
```

Comment:
Why are you calling the functions hide and show ? Why are you not calling on-click for contact button ?